packages/visual-dev/src/components/Dropdown2/Dropdown2.tsx

```diff
@@ -0,0 +1,326 @@
+        {...rest}
+      >
+        {icon && (
+          <IconView
+            color="inherit"
+            size="16px"
+            icon={icon}
+            customCSS={{
+              marginRight: "var(--sq-spacing-x-small)",
+            }}
+          />
+        )}
+        {children || placeholder}
+        <IconView
+          size="14px"
+          icon="chevron_down"
+          customCSS={{
+            marginTop: "var(--sq-spacing-xxx-small)",
+            marginLeft: "var(--sq-spacing-x-small)",
+          }}
+        />
+      </ButtonView>
+    );
+  }
+);
+
+const DropdownItemDiv = styled("div")<Required<StyleProps>>`
+  ${Styles.ItemDiv}
+  ${(props) => props.customCSS}
+`;
+
+const ItemTitleContainerDiv = styled("div")`
+  ${Styles.ItemTitleContainerDiv}
+`;
+
+const ItemSideDescriptionSpan = styled("span")`
+  ${Styles.ItemSideDescriptionSpan}
+`;
+
+const ItemDescriptionP = styled("p")`
+  ${Styles.ItemDescriptionP}
+`;
+
+const ItemView = React.forwardRef<React.ElementRef<"div">, DropdownItemProps>(
+  (props, forwardedRef) => {
+    const {
+      onClick,
+      children,
+      checked,
+      description,
+      sideDescription,
+      customCSS = {},
+      ...rest
+    } = props;
+
+    return (
+      <DropdownItemDiv
+        onClick={onClick}
+        {...rest}
+        ref={forwardedRef}
+        customCSS={customCSS}
+      >
+        <ItemTitleContainerDiv>
+          {checked && <IconView icon="checkmark" size="18px" />}
+          {children}
+          {sideDescription && (
+            <ItemSideDescriptionSpan>{sideDescription}</ItemSideDescriptionSpan>
+          )}
+        </ItemTitleContainerDiv>
+        {description && <ItemDescriptionP>{description}</ItemDescriptionP>}
+      </DropdownItemDiv>
+    );
+  }
+);
+
+const SublistDiv = styled("div")<Required<StyleProps>>`
+  ${Styles.SublistDiv}
+  ${(props) => props.customCSS}
+`;
+
+const DropdownSubItemDiv = styled("div")`
+  ${Styles.SubItemDiv}
+`;
+
+const DropdownSublistDiv = styled("div")`
+  ${Styles.DropdownSublistDiv}
+`;
+
+const SublistView = React.forwardRef<
+  React.ElementRef<"div">,
+  DropdownSublistProps
+>((props, forwardedRef) => {
+  const { name, children, customCSS = {}, ...rest } = props;
+
+  return (
+    <SublistDiv {...rest} ref={forwardedRef} customCSS={customCSS}>
+      <DropdownSublistDiv>{name}</DropdownSublistDiv>
+      <DropdownSubItemDiv>{children}</DropdownSubItemDiv>
+    </SublistDiv>
+  );
+});
+
+const DropdownNamespace = Object.assign(Dropdown2View, {
+  SublistView: SublistView,
+  ItemView: ItemView,
+  HandleView: HandleView,
+});
+
+/**
+ * @deprecated use {@link Dropdown2View} instead
+ */
+const DropdownNamespaceDeprecated = Object.assign(Dropdown2View, {
+  Sublist: SublistView,
+  ItemView: ItemView,
+  HandleView: HandleView,
+});
+
+export { DropdownNamespace as Dropdown2View };
+
+/**
+ * @deprecated use {@link Dropdown2View} instead
+ */
+export { DropdownNamespaceDeprecated as Dropdown2 };
```

Comment:
Since we never exported `Dropdown2` we don't need this deprecation warning. Just only export `Dropdown2View`